cypher/cypher-docs/src/docs/dev/query-tuning-indexes.adoc

```diff
@@ -70,6 +70,20 @@ In summary, `TEXT` indexes support the following predicates:
 |substring search| `CONTAINS`
 |===
 
+In some cases it is not possible for the system to determine an expression to be of type string.
```

Comment:
```suggestion
In some cases, the system cannot determine an expression of a type string.
```

Reply:
That does not cover what this should express. What do you think about:

```suggestion
In some cases, the system cannot determine whether an expression is of type string.
```

Reply:
Sounds good to me, yes.